tools/app/app/components/AppToolbar.tsx

```diff
@@ -34,6 +34,7 @@ export default function AppToolbar({ onNewCard }: AppToolbarProps) {
       </Box>
       <Box sx={{ flexGrow: 1 }} />
       <Button
+        id="createNewCardButton"
```

Comment:
I think it would be beneficial to consider making the references to all components either by aria-labels/text.

Reply:
Hmm, can you elaborate? I initially referred to the components via text/content but that can change in the future (or if language changed) which would then break the tests. And shouldn't aria-labels be also human-readable for screen readers and such?

Reply:
https://docs.cypress.io/guides/references/best-practices#Selecting-Elements

Actually, cypress seems to encourage to use data attributes

Reply:
Yes, that makes sense, I'll change to using data-cy for this!

Reply:
Fixed in 9a9a35b